### PR TITLE
LOAD-3218: Add additional assert paths and fix bug in creating an assert

### PR DESCRIPTION
**Pull request by paulsgrudups**

### CODE REVIEW

File: src/main/java/com/loadero/model/FileParams.java [hunk 1]
```diff
@@ -172,6 +174,10 @@ public FileParamsBuilder withPassword(String password) {
          * @return Built {@link FileParams} object.
          */
         public FileParams build() {
+            if (StringUtil.empty(this.name)) {
```
Comment (ajpetersons) [inline src/main/java/com/loadero/model/FileParams.java:177]: for other types i see this only being done in the `withName` functions. either there is something i'm missing or this should be done globally (and then in a separate task)

File: src/main/java/com/loadero/http/TestTypeAdapter.java [hunk 2]
```diff
@@ -93,8 +93,8 @@ public void write(JsonWriter jsonWriter, Test test) throws IOException {
         jsonWriter.beginObject();
         jsonWriter.name(NAME).value(test.getName());
         jsonWriter.name(SCRIPT).value(test.getScript());
-        jsonWriter.name(START_INTERVAL).value(test.getStartInterval().toString());
-        jsonWriter.name(PARTICIPANT_TIMEOUT).value(test.getParticipantTimeout().toString());
+        jsonWriter.name(START_INTERVAL).value(test.getStartInterval().getSeconds());
```
Comment (ajpetersons) [inline src/main/java/com/loadero/http/TestTypeAdapter.java:96]: may need to add reference to this MR in the GH issue

File: src/main/java/com/loadero/model/ParticipantParams.java [hunk 3]
```diff
@@ -80,7 +80,6 @@ public static ParticipantParamsBuilder builder() {
     }
 
     // Can change: name, count, location, network, browser, record_audio, computeUnit
-    // media_type
```
Comment (ajpetersons) [inline src/main/java/com/loadero/model/ParticipantParams.java:83]: but not audio and video feeds?

File: src/main/java/com/loadero/http/TestTypeAdapter.java (hunk 2 above)
Comment (paulsgrudups) [reply src/main/java/com/loadero/http/TestTypeAdapter.java:96]: Added a comment there

**Review by ajpetersons — APPROVED** [review]

**Pull request merged**